src/v2/view-builder/views/email/ChallengeAuthenticatorDataEmailView.js

```diff
@@ -0,0 +1,39 @@
+import { loc } from 'okta';
+import BaseAuthenticatorEmailView from './BaseAuthenticatorEmailView';
+
+const BaseAuthenticatorEmailForm = BaseAuthenticatorEmailView.prototype.Body;
+
+const Body = BaseAuthenticatorEmailForm.extend(
+  {
+    title() {
+      return loc('oie.email.mfa.title', 'login');
+    },
+
+    save() {
+      return loc('oie.email.verify.primaryButton', 'login');
+    },
+
+    initialize() {
+      BaseAuthenticatorEmailForm.prototype.initialize.apply(this, arguments);
+      const userEmail = this.options.appState.get('currentAuthenticatorEnrollment')?.profile?.email;
+      const subtitleText = loc('oie.email.verify.subtitle', 'login');
+
+      this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">${subtitleText}
+        <span class="strong no-translate">${userEmail}</span>
```

Comment:
The `userEmail` is not getting wrapped properly.. Make sure wherever it get displayed we try to show the full email address in one line most of the time..

Reply:
would it be fine to wrap email with block element like `p` that would always display it on the next line?

Reply:
let's double check with UX before adding something extra for this, because we have some other views that show an email address, so we should also be consistent (which is what you have right now)

Reply:
yeah let's check if this make sense to UX.. IMO it's better than existing.

Reply:
@vincentvoong-okta which one of these views should we go with?
![Screen Shot 2021-11-10 at 3 09 57 PM](https://user-images.githubusercontent.com/91922055/141212137-dc3f792e-a6c1-4c6c-8e4b-eefbf8a970e8.png)

![Screen Shot 2021-11-09 at 11 51 24 AM](https://user-images.githubusercontent.com/91922055/141212190-1a14d473-f909-4e32-8c2c-f22c9355c443.png)

Reply:
PM confirmed offline to have the appearance consistent with other screen. That is, depending on the available space the email will we either displayed as full in the same/next line or some of it will be wrapped into next line.